### DISCUSSION TITLE
Failed to start podman-clean-transient.service: Unit boot-complete.target not found.

### DISCUSSION THREAD

**Discussion by cwrau** [containers/podman#23716] [Q&A] [answered]
I found the service `podman-clean-transient.service` and having the description

```
This service runs once each boot to remove potential leftover
container state from previous boots.

This is needed when using transient storage mode in podman where the
database and other configs are stored in tmpfs, but some other files
are not. If we don't run this after an unclean boot then there may
be some leftover files that grow over time.
```

I found it good to be enabled, but starting it I got the error from the title; 

```
Failed to start podman-clean-transient.service: Unit boot-complete.target not found.
```

How is this supposed to work? The user scope doesn't have this target, it's only available for the system scope.

Or am I missing something?

**Comment by Luap99** [comment, marked as answer]
Sounds like a bug as the unit only works as root.

But are you actually using podman in transient mode? If not you do not need the unit.

**Reply by cwrau** [reply]
> But are you actually using podman in transient mode? If not you do not need the unit.

It would've been smart to read until the end 🤣
I don't think so, at least I've never activated it 😅
I just read `This service runs once each boot to remove potential leftover
container state from previous boots.` and thought "yeah, sounds good"

> Sounds like a bug as the unit only works as root.

But for those who are using that, yes, sounds like a bug, should I open an issue?

**Reply by Luap99** [reply]
> But for those who are using that, yes, sounds like a bug, should I open an issue?

Yes sure
